WilliamsGarcia/30-08/HackerRank/HashTable/HashTableExercise.java

```diff
@@ -0,0 +1,42 @@
+package com.bootcamp.daythree.hackerrank.hashtable;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+
+public class HashTableExercise {
+
+    public static final String WHITE_SPACES = "\\s+$";
+    public static final String SPACE = " ";
+    public static final String WITHOUT_SPACE = "";
+    public static final String NO = "No";
+
+    public static void main(String[] args) throws IOException {
+        
```

Comment:
Why do you put a blank space between each line? it is recommended only when a decision structure strat o finish but not in all the rows